cli/remotebazel/remotebazel.go

```diff
@@ -54,6 +54,7 @@ const (
 var (
 	execOs            = flag.String("os", "linux", "If set, requests execution on a specific OS.")
 	execArch          = flag.String("arch", "amd64", "If set, requests execution on a specific CPU architecture.")
+	containerImage    = flag.String("container_image", "", "If set, requests execution on a specific runner image. Otherwise uses the default hosted runner version.")
```

Comment:
nit: add to the help description that docker/OCI images require a `docker://` prefix